backend/grant/proposal/views.py

```diff
@@ -96,8 +96,11 @@ def post_proposal_comments(proposal_id, comment, parent_comment_id):
 
     # Make sure user has verified their email
     if not g.current_user.email_verification.has_verified:
-        message = "Please confirm your email before commenting."
-        return {"message": message}, 401
+        return {"message": "Please confirm your email before commenting"}, 401
+
+    # Make sure user is not silenced
+    if g.current_user.silenced:
+        return {"message": "User is silenced, cannot comment"}, 403
```

Comment:
Most of the other messages refer to you directly (as seen above), not indirectly. I think it might be a little less robotic to say "Your account has been silenced" instead of "User is silenced".